src/bentoml/_internal/cloud/deployment.py

```diff
@@ -24,7 +91,7 @@ def list_deployment(
             raise BentoMLException("List deployments request failed")
         return res
 
-    def create_deployment(
+    def create_deployment_by_json(
```

Comment:
```suggestion
    def create_deployment_json(
```

Reply:
lets accept dict instead of a string like this. It is an anti pattern

Reply:
If you want to parse from a file, then you can have a `create_deployment_from_file`.